VimeoNetworking/Sources/Models/VIMVideo.m

```diff
@@ -506,4 +506,19 @@ - (BOOL)isLive
     return self.live != nil;
 }
 
+- (BOOL)isPreBroadcast
+{
+    return self.isLive && ([self.live.status isEqual: VIMLive.LiveStreamStatusUnavailable] || [self.live.status isEqual: VIMLive.LiveStreamStatusReady] || [self.live.status isEqual: VIMLive.LiveStreamStatusPending]);
+}
+
+- (BOOL)isMidBroadcast
+{
+    return self.isLive && [self.live.status isEqual: VIMLive.LiveStreamStatusStreaming];
+}
+
+- (BOOL)isPostBroadcast
+{
+    return self.isLive && ([self.live.status isEqual: VIMLive.LiveStreamStatusDone] || [self.live.status isEqual: VIMLive.LiveStreamStatusStreamingError]);
```

Comment:
Per our conversation, let's make sure that checking for `streaming_error` is relevant for post-broadcasting.

Reply:
Ah, I see what you're saying now. That sounds good.